### PR TITLE
Fix golint errors for pkg/kubelet/dockershim/network/kubenet

### PR DESCRIPTION
**Pull request by shimmy568**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

This PR fixed linting issues in `pkg/kubelet/dockershim/network/kubenet`

**Which issue(s) this PR fixes**:

Part of #68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

/priority backlog
/area code-organization

### CODE REVIEW

File: pkg/kubelet/dockershim/network/kubenet/kubenet.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
 package kubenet
 
+// Defining constants for package
```
Comment (mattjmcnaughton) [inline pkg/kubelet/dockershim/network/kubenet/kubenet.go]: Can you share more about what why `Golint` requires this comment? I feel the `// Defining constants for package` doesn't provide any meaningful additional info for the reader.

Would Golint also stop complaining if we instead gave each individual constant a descriptive comment?

File: pkg/kubelet/dockershim/network/kubenet/kubenet_linux.go [hunk 2]
```diff
@@ -51,6 +51,7 @@ import (
 	netutils "k8s.io/utils/net"
 )
 
+// Define constants
```
Comment (mattjmcnaughton) [inline pkg/kubelet/dockershim/network/kubenet/kubenet_linux.go]: Same comment as above :)

**Review by mattjmcnaughton — COMMENTED** [review]
Looks good aside from my one question around adding comments for constants.

/ok-to-test

File: pkg/kubelet/dockershim/network/kubenet/kubenet.go (hunk 1 above)
Comment (shimmy568) [reply pkg/kubelet/dockershim/network/kubenet/kubenet.go]: Golint just wants a comment for exported fields. But you're right adding one comment for the whole block doesn't really help very much. I'll update with more descriptive comments for the constants.

**Comment by shimmy568** [comment]
Accidentally did a amend commit which made a merge when push the change. Seems to be all good now, has the changes with the updated comments as well.

**Comment by xichengliudui** [comment]
good job
/lgtm

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm

Thanks for taking the time to ensure all comments provide a benefit to the reader.

**Comment by danwinship** [comment]
@shimmy568 sorry this PR got forgotten about... can you rebase it to current master and re-push?

**Pull request closed without merging**